### PR TITLE
Prepare new website/docs release

### PR DESCRIPTION
**Pull request by peterszerzo**

### CODE REVIEW

File: packages/website/src/index.tsx [hunk 1]
```diff
@@ -119,17 +119,20 @@ export default class App extends React.Component<{}, {}> {
             </Hero>
             <StaticContent
               css={{ marginTop: 32 }}
-              markdownContent={`\`Operational UI\` is a library made for data-driven decision making. It does its best when used for interfaces that assume familiarity through routine use, prioritizing compactness and subtle hierarchies. 
+              markdownContent={`\`Operational\` is a UI library made that caters to day-to-day operational decision-makig. It does its best when used for interfaces that assume familiarity through routine use, prioritizing compactness and [small effective differences](https://twitter.com/edwardtufte/status/450076034759524352).
```
Comment (TejasQ) [inline packages/website/src/index.tsx]: Why is ``Operational`` in backticks?

Comment (TejasQ) [reply packages/website/src/index.tsx]: Also, `decision-makig`.

Comment (peterszerzo) [reply packages/website/src/index.tsx]: It looks like code that way, purely for the `aesthetic kicks` :)

**Comment by peterszerzo** [comment]
The typo has been fixed, and the branch is ready for merge, @TejasQ. I'll be updating it as I go through visual improvements in a separate branch.

File: packages/components/src/Message.tsx [hunk 2]
```diff
@@ -13,18 +13,17 @@ export interface Props {
...
+  const backgroundColor = expandColor(theme, color) || theme.colors.info
   const textColor = readableTextColor(backgroundColor, [theme.colors.black, "white"])
 
   return {
     backgroundColor,
     overflow: "hidden",
     color: textColor,
     padding: `${theme.spacing * 1 / 2}px ${theme.spacing}px`,
-    borderRadius: 2,
+    borderRadius: 4,
```
Comment (TejasQ) [inline packages/components/src/Message.tsx]: Should this be a theme property?

Comment (peterszerzo) [reply packages/components/src/Message.tsx]: Yepp, coming up in a different branch!

**Pull request merged**